test/tpch/postgres-bench/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM postgres:latest
+
+COPY postgres-entrypoint.sh /usr/local/bin/postgres-entrypoint.sh
+RUN chmod +x /usr/local/bin/postgres-entrypoint.sh
+
+COPY backup_*.sql /sql_files/
+
+ENTRYPOINT ["/usr/local/bin/postgres-entrypoint.sh"]
```

Comment:
Instead of doing the work to import the TPCH files everytime we start the postgres server, can't we bake in an image that already has the databases loaded so we just pull it down and its ready to go?

Reply:
Nope, the only way to get populated data in an image is to have sql dump with base image. That's why Postgres and MySQL are both using this approach.